src/screens/Post/PostContentScreen.tsx

```
@@ -267,35 +271,40 @@ function PostContentScreen({route}: any): JSX.Element {
+                  icon={'smile'}
+                  size={theme.sizes.medium}
+                  color={
+                    user?.$id &&
+                    postMetrics &&
+                    postMetrics.likes.indexOf(user?.$id) > -1
+                      ? theme.colors.positive
+                      : theme.colors.text_color
+                  }
+                />
+                <CustomText
+                  title={postMetrics && postMetrics.likes.length.toString()}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)